pinot-core/src/test/java/org/apache/pinot/core/common/datatable/DataTableSerDeTest.java

```diff
@@ -535,56 +574,207 @@ public void testDataTableMetadataBytesLayout()
     }
   }
 
+  private RowDataBlock getDataBlockFromRandomData(DataSchema schema, int numColumns)
```

Comment:
I don't think this is needed. We should use `fillDataTableWithRandomData()`

Reply:
I added this to be able to test: `DataBlockBuilder.buildFromRows`. Will remove.